plugin/src/main/java/hudson/plugins/findbugs/FindBugsPublisher.java

```diff
@@ -154,6 +164,21 @@ public boolean isRankActivated() {
     }
 
     /**
+     * Added to properly uncoercing.
+     */
+    public boolean getIsRankActivated() {
+        return isRankActivated;
```

Comment:
the method name is wrong.   the return is a boolean so it needs to start with `is` regardless of if you change the name of the bean property